pages/index.tsx

```diff
@@ -78,6 +81,7 @@ function IndexPage() {
                         </a>
                     </small>
                 </div>
+                <ToggleSwitch onClick={() => setIsWowWowMode(!isWowWowMode)} active={isWowWowMode} />
```

Comment:
handler`() => setIsWowWowMode(!isWowWowMode)`をuseCallbackで包んで渡してほしいです
```suggestion
                <ToggleSwitch onClick={handleClick} active={isWowWowMode} />
```